projects/packages/search/src/instant-search/lib/errors.js

```diff
@@ -0,0 +1,38 @@
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Returns an error message based on the error code.
+ *
+ * @param {Error} error - Error object
+ * @returns {*}	an error message
+ */
+export function getErrorMessage( error ) {
+	switch ( error?.message ) {
+		case 'service_unavailable':
+			return __(
+				'Jetpack Search is currently unavailable. Please try again later.',
+				'jetpack-search-pkg'
+			);
+		case 'unknown_blog':
+		case 'unauthorized':
+			return __( 'You are not authorized to perform search on the website.', 'jetpack-search-pkg' );
```

Comment:
```suggestion
			return __( 'You are not authorized to perform a search on the website.', 'jetpack-search-pkg' );
```
Grammarly says there should be an `a`. 😄

Reply:
@dognose24 it's funny that the error message was suggested by copilot 😂 Why you bots fight one another, you should be like brothers 🥹

Changed the wording to 'You are not authorized to search on the website.'.